server/lib/statsd-collector.js

```diff
@@ -65,10 +75,34 @@ function getGenericTags(body) {
     }
   }
 
+  // collect AB choices
+  if (body.ab) {
+    var abTags = [];
+
+    body.ab.forEach(function (item) {
+      if (item.event && item.event === 'choice' && item.experiment && item.choice) {
+        var tagName = 'ab_' + _normalizeTag(item.experiment);
+        var hadChoice = false;
+
+        Object.keys(item.choice).forEach(function (choiceKey) {
+          hadChoice = true;
+          tagName += '_' + _normalizeTag(choiceKey);
```

Comment:
Yeah we cannot do `&` values, see more at http://docs.datadoghq.com/guides/dogstatsd/#metrics-1 the DataDog graph ui will not support it.